kolibri/core/assets/src/views/core-base.vue

```diff
@@ -121,15 +123,15 @@
 
   @require '~kolibri.styles.definitions'
 
-  .loading-spinner-fixed
-    position: fixed
+  .align-to-parent {
+    position: absolute
+    top: 0
+    left: 0
+  }
```

Comment:
forgot to mention this before, but this is throwing a lint error - we disallow curly brackets in Stylus.

You can check for this by running with the `--lint` flag:

```kolibri manage devserver --debug -- --webpack --lint --qcluster```